app/controllers/users_controller.rb

```diff
@@ -10,23 +10,24 @@ def new
     @user = User.new(user_params)
   end
 
-  def create
-    binding.pry
+  def auth
     auth_hash = request.env['omniauth.auth']
     email = auth_hash['info']['email']
     user = User.find_or_create_by(email: email)
 
     session[:access_token] = auth_hash['credentials']['token']
     redirect_to dashboard_path
   end
-    # @user = User.new(user_params)
-    # if @user.save
-    #   session[:user_id] = @user.id
-    #   redirect_to dashboard_path(@user.id)
-    # else
-    #   render :new
-    # end
-  # end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
This create action is similar to what I was thinking, but from the registration form it then creates the user account but with additional user params that aren't currently in our code on main, such as address, interests, etc... all the columns in the user table.